ui-guidelines/css/resources.css

```diff
@@ -0,0 +1,128 @@
+/*==== TYPOGRAPHY ====*/
+@import url(https://fonts.googleapis.com/css?family=Open+Sans);
+
+.font-weights {
+	width: 100%;
+	height: 10em;
+}
+.weight-sample {
+	width: 20%;
+	float: left;
+	margin-right: 5%;
+	border: 1px solid #fce3da;
+	border-radius: 8px;
+}
+.big-letter{
+	font-size: 6em;
+	font-family: 'Open Sans', Arial, sans-serif;
+}
+.light {font-weight: 300;}
+.normal {font-weight: 500;}
+.medium {font-weight: 600;}
+.bold {font-weight: 700;}
+.text-size {margin-left: 1.5em;}
+
+.typography, .typography p {
+	font-family: 'Open Sans', Arial, sans-serif;
+}
+
+.text-h1 {font-size: 2.5em;}
+.text-h2 {font-size: 1.5em;}
+.text-h3 {font-size: 1.25em;}
+
+.text-x-huge {font-size: 2.5em;}
```

Comment:
Still, I'm not sure we need that much font sizes in our project. We should reduce the amount of them.